Andlet/Andlet/Views/Forms/Step1View.swift

```diff
@@ -1,102 +1,216 @@
 import SwiftUI
-import PhotosUI
 
 struct Step1View: View {
```

Comment:
Taking into account that you deleted the FormView now this will be the view that the other views should call ?

Reply:
Exactly! We love simplicity :)